inst/rmd/h/default.html

```diff
@@ -51,15 +51,23 @@
 <style type="text/css">code{white-space: pre;}</style>
 <style type="text/css">
 $highlighting-css$
+
 $if(code_folding)$
-div.sourceCode {
-  overflow-x: visible;
+
+.row {
+  width: 100%;
 }
-$else$
-div.sourceCode {
-  overflow-x: auto;
+
+.sourceCode {
+  overflow: auto;
+}
+
+.code-folding-btn {
+  margin-right: -29px;
```

Comment:
I see what you mean by `-29px`, but some people's eyes are really good (e.g. our CTO) and may notice the 1px distance to the right margin :)

Reply:
This is now `-30px`! Not sure why my eyes thought `29px` was better, `30px` makes logical sense.